scripts/format.sh

```diff
@@ -1,16 +0,0 @@
-#!/bin/sh
```

Comment:
Ah. That is it. I added `format.sh` it with `-f`. remove ignore from `.gitignore` is great.